src/install/requirements_common.txt

```diff
@@ -7,6 +7,7 @@ setuptools
 appdirs==1.4.4
 flaky==3.7.0
 lief==0.12.3
+matplotlib==3.5.3
```

Comment:
Grepping (`rg matplotlib`) through the code reveals that matplotlib is only ever used in `src/helperFunctions/web_interface.py` which would only be frontend.

I don't know why it is listed as backend dependency, but I think it should not be.

Reply:
I don't know but possibly because of the plugin